examples/Procfile

```diff
@@ -0,0 +1,3 @@
+build-bar: yarn watch:mobile
+serve-bar: cd dist; python -m SimpleHTTPServer 8000
```

Comment:
This doesn't work with Python3 since we don't use `SimpleHTTPServer` but `http.server` instead

Reply:
I updated the Procfile with `python3 -m http.server`, thanks for the tip :)